Source/PeleLM_derive.cpp

```diff
@@ -43,41 +51,49 @@ void pelelm_derdvrho (const Box& bx, FArrayBox& derfab, int dcomp, int /*ncomp*/
 
 
 //
-// Extract species mass rhoY_n
+// rhoY_n, identity ---------- FIXME?
```

Comment:
What has to be fixed?

Reply:
Because this does not require a derive.  It's an identity copy and is not necessary.